frontend/test/metabase/scenarios/dashboard/dashboard-drill.cy.spec.js

```diff
@@ -164,6 +170,76 @@ describe("scenarios > dashboard > dashboard drill", () => {
       .parent()
       .within(() => cy.findByText("foo"));
   });
+
+  it("should pass multiple filters for numeric column on drill-through (metabase#13062)", () => {
+    // go to admin > data model > sample dataset > reviews
+    cy.visit("/admin/datamodel/database/1/table/4");
+
+    // Set "Rating" Field type to: "Category" ("Score" is selected by default)
+    cy.findByText("Score").click();
+    // "Category" is not visible and any other method couldn't find it, including `Popover().contains("Category")`
+    cy.get(".ReactVirtualized__Grid")
+      .scrollTo("top")
+      .contains("Category")
+      .click();
+    // make sure the field updated before navigating away
+    cy.findByText("Category");
+
+    // go straight to simple question > reviews
+    cy.visit("/question/new?database=1&table=4");
+
+    // save the question
+    cy.findByText("Save").click();
+    cy.get(".Modal").within(() => {
+      cy.findByText("Save").click();
+    });
+    // and add it to a new dashboard
+    cy.findByText("Yes please!").click();
+    cy.findByText("Create a new dashboard").click();
+    cy.findByLabelText("Name")
+      .click()
+      .type("13062");
+    cy.findByText("Create").click();
+
+    // make sure we switched to the dashboard in edit mode
+    cy.findByText("You're editing this dashboard.");
+
+    // add filter
+    cy.get(".Icon-filter").click();
+    cy.findByText("Other Categories").click();
+    // and link it to the card
+    selectDashboardFilter(cy.get(".DashCard"), "Rating");
+
+    // save the dashboard and exit editing mode
+    cy.findByText("Save").click();
+    cy.findByText("You're editing this dashboard.").should("not.exist");
```

Comment:
Not a big deal, but I think it'd be good for us to move towards doing this setup through the API rather than the UI.

We still need a lot more helpers to make that easy though. e.g. a helper to update the field via the API could save ten lines in this test

Reply:
I was thinking about it. Not just for this test, but for a lot of others. I should start slowly converting a lot of "manual" work into programatic/API approach. I feel as if I need a bit more time and experience to be confident in taking that step.
Regardless, I'd be super happy to discuss this further whenever you have time.